components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIManager.java

```diff
@@ -894,8 +894,8 @@ String addProductResourceFile(APIProductIdentifier identifier, String resourcePa
      * @return
      * @throws APIManagementException
      */
-    Map<String, Object> searchPaginatedAPIs(String searchQuery, String organization, int start, int end)
-            throws APIManagementException;
+    Map<String, Object> searchPaginatedAPIs(String searchQuery, String organization, int start, int end,
+            String sortBy, String sortOrder) throws APIManagementException;
```

Comment:
define an `enum` for the `sortOrder` is better than a `String`.
there are 2 possible values for sort order (Ascending or Descending). 
So enums `sortOrder.ASC` and `sortOrder.DESC` would fine.

Reply:
Rest api call for /apis resorce will look like this after the choreo console fix is done: `https://app.choreo.dev/apim/proxy/api/am/publisher/v2/apis?organizationId=4f80b73c-1552-42ef-b4cd-59ca3e312ddd&offset=10&limit=5&sortBy=createdTime&sortOrder=desc`
So in there `sortOrder` is passing as a string.
@malinthaprasan Do we need to get the `sortOrder` query parameter as an enum?

Reply:
Defined an enum for `sortOrder` schema: https://github.com/wso2/carbon-apimgt/pull/10767/commits/0d7ea12a9c5a38561d38b86eab8548d04a914864